docs/package.json

```diff
@@ -69,11 +48,7 @@
     "style-loader": "^0.23.1",
     "styled-components": "^3.4.10",
     "ts-node": "^9.1.1",
-    "typescript": "^4.1.3",
-    "unfetch": "^3.0.0",
-    "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.1",
-    "webpack-dev-server": "^3.3.1"
```

Comment:
I think since these dependencies are only used by the `docs` package they should remain in the `docs` package and be removed from the root package.